ovirt-imageio.spec.in

```diff
@@ -13,17 +13,41 @@ Source0:   https://github.com/oVirt/%{name}/releases/download/v%{version}/%{name
 %global admin_confdir %{_sysconfdir}/%{name}
 %global vendor_confdir %{_prefix}/lib/%{name}
 
+# We need to have Python 3.8 support of imageio client for oVirt Ansible
+# Collection running with ansible-core-2.12, which uses non-standard Python 3.8
+# on EL8 platform
+%if "0%{?rhel}" == "08"
```

Comment:
Why compare to "08" instead of "8"?

Reply:
We have tried in other projects `%if 0%{?rhel} == 8` and it didn't work, so I've tried to compare strings and it works now. But maybe it could work even without 0

Reply:
Fixed